### ISSUE TITLE
How to ground truth cell masks into bounding boxes

### ISSUE THREAD

**Issue by iamyufan** [vanvalenlab/cellSAM#19]
Hi,

Thank you for your amazon work! I know that you are working on the training scripts for CellSAM. But I do have one quick question: how do you convert the ground truth cell mask labels to bounding boxes in the step of training CellFinder? Is there any related code in the repo or it will be included in the published training scripts?

Thank you in advance!
Yufan

**Comment by damaggu** [comment]
Hi Yufan,

Thanks for your question! We will soon release training code with examples for training dataset generation. Stay tuned this week/next week for updates, and please follow up with any other questions you have based on those examples.

Very best!
